plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.tsx

```diff
@@ -33,13 +36,33 @@ function ProductCard( props: ProductCardProps ): JSX.Element {
 		vendorName: '',
 		vendorUrl: '',
 		icon: '',
+		label: null,
+		primary_color: null,
```

Comment:
Any reason to use `primaryColor` instead of `primary_color` for consistency?

Reply:
The object gets `primary_color` as standard (that's what comes from the API, for some reason). I initially remapped it with `primaryColor: product.primary_color` but I reasoned that this was actually worse because we were now using _two_ different names for the same thing!

Consistency within the variable seemed to be more-important than consistency between different variables.